controllers/workspace/devworkspace_controller.go

```diff
@@ -394,3 +307,35 @@ func getWorkspaceId(instance *devworkspace.DevWorkspace) (string, error) {
 	}
 	return "workspace" + strings.Join(strings.Split(uid.String(), "-")[0:3], ""), nil
 }
+
+func (r *DevWorkspaceReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// TODO: Set up indexing https://book.kubebuilder.io/cronjob-tutorial/controller-implementation.html#setup
```

Comment:
Should we create issues for these improvements

Reply:
I'll create some issues on Monday, that's a good point.

Reply:
Created #192 and #191